metacat-common/src/test/java/edu/ucsb/nceas/metacat/common/query/SolrQueryServiceControllerTest.java

```diff
@@ -47,11 +50,12 @@ public void testGetSolrSpecVersion() throws UnsupportedType, NotFound,
      */
     @Test
     public void testgetIndexSchemaFields() throws Exception {
-       Map<String, SchemaField> fields = SolrQueryServiceController
+        int targetSize = Settings.getConfiguration().getInt("solr.schemField.size");
+        Map<String, SchemaField> fields = SolrQueryServiceController
                                                        .getInstance().getIndexSchemaFields();
-       assertTrue(fields != null);
-       assertTrue("The number of index schema fields should be 165 rather than "
-                   + fields.size(), fields.size() == 165);
+        assertTrue(fields != null);
+        assertTrue("The number of index schema fields should be " + targetSize + " rather than "
```

Comment:
why not `assertEquals`? (That would automatically print out the `Expected` and `Actual` values if they differ, without you needing to concatenate them into the string message.) i.e.:

```java
assertEquals("The number of index schema fields is incorrect", targetSize, fields.size());
```